Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,41 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
+
+    if build.with?("tex") && (ps2pdf = which("ps2pdf")).nil?
+      saved_path = ENV["PATH"]
+      ENV.append_path "PATH", "/usr/local/bin" # known install location by MacTeX
+      odie "Couldn't find ps2pdf" if (ps2pdf = which("ps2pdf")).nil?
+      ENV["PATH"] = saved_path
     end
 
-    system "./configure", "-prefix", prefix, "-mandir", man, strictness
-    # this build fails if jobs are parallelized
-    ENV.deparallelize
+    system "./configure", *args
     system "make", "world.opt"
     system "make", "install"
+
+    if build.with?("doc") || build.with?("tex")
+      ENV.prepend_path "PATH", Formula["gnu-sed"].opt_libexec/"gnubin"
+      ENV.deparallelize
+      cd "doc/htmlp"
+      if build.with? "doc"
+        system "make"
+        doc.install "../html", Dir["camlp5.info*"]
+      end
+      if build.with? "tex"
+        inreplace "Makefile", "ps2pdf", ps2pdf
```

Comment:
The issue is here. The geniuses who put together MacTeX didn't install ps2pdf in /Library/TeX/texbin but rather in /usr/local/bin.

I could add a whole additional requirement just for :ps2pdf but since most people get their TeX from MacTeX, that seemed like it would be overkill.